lib/ioki/http_adapter.rb

```diff
@@ -21,7 +21,7 @@ def self.get(config)
           f.response :logger, config.logger, config.logger_options do |logger|
             logger.filter(/(X-Client-Secret: ")([^"]+)/, '\1[REMOVED]')
             logger.filter(/(Authorization: ")([^"]+)/, '\1[REMOVED]')
-            logger.filter(/("token":")([^"]+)/, '\1[REMOVED]')
```

Comment:
I'm wondering if we should have both versions. AFAIK => is usually used for backwards compatibility in debug/log output. This might change at some point.

Reply:
good point, it's easy enough to have both